core/controllers/feedback.py

```diff
@@ -120,6 +120,23 @@ def get(self):
         })
 
 
+class FeedbackStatsHandler(base.BaseHandler):
+    """Returns Feedback stats for an exploration.
+        - Number of open threads
+        - Number of total threads
+    """
+    
+    def get(self, exploration_id):
+        feedback_thread_analytics = feedback_services.get_thread_analytics(
```

Comment:
Oh -- I meant, should we throw a 404? Though I guess returning (0, 0) is fine too...